src/web/src/pages/NewCourseScheduler.vue

```diff
@@ -473,6 +466,80 @@ export default {
         this.addCourse(course);
       }
     },
+    getSchedules() {
+      const oldLength = this.possibilities.length;
+      try {
+        if (Object.values(this.selectedCourses).length === 0) {
+          this.possibilities = [{
+            sections: [],
+            time: [0, 0, 0, 0, 0]
+          }];
+        }
+        const result = this.generateSchedule(Object.values(this.selectedCourses))
+        if (!result.length) {
+          throw new Error('conflict!')
+        }
+        this.possibilities = result
+        
+        //Don't set this.index to 0 if just loaded cookie
+        if (this.loadedIndexCookie == 2) {
```

Comment:
if we're generating the new schedules here, shouldn't we always reset the index to 0?

Reply:
If we just always set the index to 0, when the index cookie is loaded, it will load the saved index but then set it back to 0 so the index won't be the one from the cookie